<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizLocation</class>
 <widget class="QWidget" name="WizLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>383</height>
   </rect>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QLabel" name="Banner">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>360</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="GeoTitle">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Choose Your Home Location</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>12</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="WelcomeText">
       <property name="text">
        <string>&lt;html&gt;&lt;head&gt;&lt;/head&gt;&lt;body&gt;&lt;p&gt;Select a City near your location from the list. You may filter the list by the name of your city, province, and country.&lt;/p&gt;&lt;p&gt;Once you have selected a City, press &lt;span style=&quot; font-weight:600;&quot;&gt;Next&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="CountryLabel">
         <property name="text">
          <string>Country:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="CityFilter">
         <property name="toolTip">
          <string>Filter the list by city name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="CityLabel">
         <property name="text">
          <string>City:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ProvinceLabel">
         <property name="text">
          <string>Province:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="CountryFilter">
         <property name="toolTip">
          <string>Filter the list by country name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="ProvinceFilter">
         <property name="toolTip">
          <string>Filter the list by province name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="1">
        <widget class="dmsBox" name="LongBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LatLabel">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="LongLabel">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="dmsBox" name="LatBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="CityListBox" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>The list of cities which match the present search filters.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CityFilter</tabstop>
  <tabstop>ProvinceFilter</tabstop>
  <tabstop>CountryFilter</tabstop>
  <tabstop>LongBox</tabstop>
  <tabstop>LatBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
